vm/src/stdlib/os.rs

```diff
@@ -1073,8 +1082,8 @@ mod posix {
     use nix::unistd::getgroups;
 
     #[cfg(target_os = "redox")]
-    fn getgroups() -> nix::Result<Vec<Gid>> {
-        unimplemented!("redox getgroups")
+    fn getgroups(vm: &VirtualMachine) -> nix::Result<Vec<Gid>> {
+        os_unimpl("os.getgroups", vm)
```

Comment:
`on_unimpl` returns `PyResult` but this function return `nix::Result`

Reply:
Oops, nice catch